backend/sound/Mock_OSC_Client.py

```diff
@@ -0,0 +1,19 @@
+from UDPClient import ClientIO
+import time
+import random
+import pickle
+
+mock_client = ClientIO('127.0.0.1', 5020)
+categories = ['Kritik', 'Lob']
+level_values = [3, 5, 8]
+
+dict = {}
```

Comment:
`dict` also is built-in function name in python, so the same comment holds here, too. Plus, why do you declare the empty dict here? It's enough to declare it when you assign it a value (in line 14)